### PR TITLE
Fixes/improvements in executeStartScripts method +  load-examples.groovy + remove wkws residing in proactive-examples

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L238](https://github.com/ow2-proactive/scheduling/blob/6f0a628cf6933ce17f5c90a23e3e66b9d5565895/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L238): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L238](https://github.com/ow2-proactive/scheduling/blob/d1dd6a9d24a1c3bccf63c70e81a4fbbefc313804/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L238): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L238](https://github.com/ow2-proactive/scheduling/blob/54c174d0f5f60452e40dbe2d2119ffebc4af4e47/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L238): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

File: tools/load-examples.groovy [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+				def object_list = slurper.parseText(response.toString())
+				def object_found = object_list.find {object -> object.name == bucket}
+
+				println "[" + scriptFileName + "] bucket " + bucket + " found? " + (object_found != null)
+
+				// Create a bucket if needed -------------
+				def bucket_id = null
+				if (object_found)
+					bucket_id = object_found.id
+				else {
```
Comment (MaelAudren) [inline tools/load-examples.groovy]: It is considered as good practice to always put brace after a if. Maybe for single line if it is ok to not put them but here there is a else after. Could you add brace after the if ?

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

1. MINOR [SchedulerStarter.java#L238](https://github.com/ow2-proactive/scheduling/blob/45b8d7c0b62ef38510937d1694de5629a145607c/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java#L238): Define and throw a dedicated exception instead of using a generic one. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)

**Pull request merged**